autopush/main.py

```diff
@@ -63,11 +63,11 @@ class AutopushMultiService(MultiService):
 
     THREAD_POOL_SIZE = 50
 
-    def __init__(self, conf):
+    def __init__(self, conf, sessions=None):
```

Comment:
nitpicky: ideally I think DatabaseManager being more responsible for creating the BotoSessions (maybe taking a config for it that includes the endpoint_url, say for tests) would avoid the hooks here. A lot like it is responsible for the Table objects, but also allowing overrides of those objects for tests.

Unfortunately our tests around main'ish functions are currently kind of crappy and would probably require some annoying monkey patches to force the right test configuration